agent/consul/usagemetrics/usagemetrics_oss_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/hashicorp/consul/agent/consul/state"
 	"github.com/hashicorp/consul/agent/structs"
 	"github.com/hashicorp/consul/sdk/testutil"
+	"github.com/hashicorp/serf/serf"
```

Comment:
Just a small note that we will probably want to follow up with some tests in enterprise as well to ensure the right segment gets reported.

Reply:
Theoretically that could be tested in the OSS tests as well, all we'd need to add is `segment` tags to the items in the array returned by the `getMembersFunc` we test with

Reply:
oooh, even better

Reply:
Just added this to the test. Now instead of 1 client there are two, each in their own segment (`"a"` and `"b"` respectively).